pkg/infoschema/infoschema_v2.go

```diff
@@ -1456,3 +1465,14 @@ func (is *infoschemaV2) ListTablesWithSpecialAttribute(filter specialAttributeFi
 	}
 	return ret
 }
+
+type refillOption string
```

Comment:
```suggestion
type refillOption struct{}
```

no need to assign the "refillOptionKey" for it? We don't check the key string.